pokete.py

```diff
@@ -1341,7 +1000,7 @@ def gen_obs():
                       map_data[ob_map]["soft_obs"][soft_ob])
         for dor in map_data[ob_map]["dors"]:
             parse_obj(_map, dor,
-                      Dor(" ", state="float",
+                      Door(" ", state="float",
                           arg_proto=map_data[ob_map]["dors"][dor]["args"]),
                       map_data[ob_map]["dors"][dor])
```

Comment:
Also rename this variable from `dor` to `door`

```suggestion
        for door in map_data[ob_map]["dors"]:
            parse_obj(_map, door,
                      Door(" ", state="float",
                           arg_proto=map_data[ob_map]["dors"][door]["args"]),
                      map_data[ob_map]["dors"][door])
```